plugin/server/datastore-sqlite/sqlite.go

```diff
@@ -691,3 +698,25 @@ func main() {
 		GRPCServer: plugin.DefaultGRPCServer,
 	})
 }
+
+func getListSelectorQuery(conditionCount int) string {
+	query := `
+SELECT *, group_concat(selector, '') as compound 
+FROM (
```

Comment:
Does sqlite have `EXPLAIN`? :) would be good to know if there's anything we can index to make this performant. @paul-argeniss mentioned that he managed to run into selector locking issues in the stresstest... I wonder if this will address some of that

Reply:
There is ```EXPLAIN QUERY PLAN``` which I need to [read](https://sqlite.org/eqp.html) about but you can get something like the next with it:
```
"1"	"0"	"1"	"SCAN TABLE selectors"
"1"	"1"	"0"	"SEARCH TABLE registered_entries USING INDEX sqlite_autoindex_registered_entries_1 (registered_entry_id=?)"
"1"	"0"	"0"	"USE TEMP B-TREE FOR ORDER BY"
"0"	"0"	"0"	"SCAN SUBQUERY 1"
"0"	"0"	"0"	"USE TEMP B-TREE FOR GROUP BY"
``` 

My current concern is that we scan the whole selectors table to build the compound selector.